### PR TITLE
use official github action

### PR DESCRIPTION
**Pull request by tyler36**
The new image helps centralize many common steps and makes for easier maintainence:

https://github.com/ddev/github-action-add-on-test

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,74 +18,28 @@ defaults:
...
-      run: brew install ddev/ddev/ddev
-
-    - name: Use ddev edge
-      if: matrix.ddev_version == 'edge'
-      run: brew install ddev/ddev-edge/ddev
-
-    - name: Use ddev HEAD
-      if: matrix.ddev_version == 'HEAD'
-      run: brew install --HEAD ddev/ddev/ddev
+        sudo apt-get update >/dev/null && sudo apt-get install -y expect >/dev/null
```
Comment (rfay) [inline .github/workflows/tests.yml:37]: Does this need expect?

**Review by rfay — APPROVED** [review]
Looks great, thanks!

**Comment by tyler36** [comment]
I remove the line. We'll see if the tests fail. :)

**Review by stasadev — APPROVED** [review]
Thank you!

**Pull request merged**